build/implementations/get-framed-report.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
+const globby = require('globby')
+const axios = require('axios')
+const jsonld = require('jsonld')
+const frameConfig = require('./json-ld-frame-config')
+
+const getFramedReport = async options => {
+	const { type, path } = options
+
+	if (![`NPM`, `JSON`].includes(type)) {
```

Comment:
Shouldn't you be able to just tell from the path whether or not you're looking at a URL or a local file? Type seems like a redundant field.